raspberry-mint-dev/src/css/components/header.scss

```diff
@@ -0,0 +1,49 @@
+#header {
+  background-color: $mint-900;
+
+  .header-site-logo {
+    background-image: url("/betty-192x192.png");
+    background-position: center left;
+    background-repeat: no-repeat;
+    background-size: 3rem;
```

Comment:
Can we set a fixed height and a max width? If not with a background image, then with an <img>?